builtin/settingtypes.txt

```diff
@@ -892,6 +892,11 @@ emergequeue_limit_generate (Limit of emerge queues to generate) int 32
 #    at the cost of slightly buggy caves.
 num_emerge_threads (Number of emerge threads) int 1
 
+#   Number of threads to use for mapblock mesh data updae. Make this field blank,
+#   or increase this number to use multiple threads. This may speed up map display
+#   time but may also make the game slower.
+mesh_update_threads (Number of mapblock mesh update threads) int 1
```

Comment:
Maybe put it in the Client->graphics->advanced section? mapgen seems not appropriate.